### ISSUE TITLE
On History tab, group and collapse daily entries by month and year to help organisation

### ISSUE THREAD

**Issue by davidtimmsinshur** [a85/POSTMan-Chrome-Extension#8677]
**Is your feature request related to a problem? Please describe.**
I have only been using Postman around 9 months, already I have a problem. I want to keep maybe the last 2 months max of history, I definitely want to keep the last few days worth. That is a painful process to delete all those individual daily entries, to the point that I am never going to do it. It is also difficult to navigate in its current format. All items are automatically expanded.

**Describe the solution you'd like**
Group history items by day as current but add the following higher level groups in a tree structure like (a bit like the date tree you get on some blogs):

```
Today
Yesterday
{years}
    {opened to current month}
        {dates}
    {months}
        {dates}
```

Allow deletion or other actions to take place at each group level as well as on individual requests.
Allow multiple selections/deselections and commonly associated keyboard shortcuts including Ctrl-Shift-down, Ctrl-A, delete key etc

**Describe alternatives you've considered**
This would actually be great features in their own right:
- Make _Clear All_ button a _Clear items..._ dialogue then allow user to customise delete period from a drop down. (a bit like Chrome Clear Browsing History dialogue except you're keeping history).

**Comment by a85** [comment]
I agree with you David and thanks for posting this issue. We are working on improving this experience a lot.
